### PR TITLE
Disallow spaces in staging content filenames

### PR DESCRIPTION
**Pull request by davidsbailey**

### CODE REVIEW

File: tools/hooks/hooks_utils.rb [hunk 1]
```diff
@@ -11,6 +11,6 @@ def self.get_unstaged_files
 
   def self.get_staged_files
     Dir.chdir File.expand_path('../../../', __FILE__)
-    `git diff --cached --name-only --diff-filter AM`.split("\n").map(&:chomp).map {|x| File.expand_path("../../../#{x}", __FILE__)}
+    `git diff --cached --name-only --diff-filter AMR`.split("\n").map(&:chomp).map {|x| File.expand_path("../../../#{x}", __FILE__)}
```
Comment (aoby) [inline tools/hooks/hooks_utils.rb:14]: Is the [R](https://git-scm.com/docs/git-diff#git-diff--R) option the only change here? What does it do? I can't make sense of this from the docs:
> Swap two inputs; that is, show differences from index or on-disk file to tree contents.

Comment (davidsbailey) [reply tools/hooks/hooks_utils.rb:14]: from `git help diff`
```
       --diff-filter=[(A|C|D|M|R|T|U|X|B)...[*]]
           Select only files that are Added (A), Copied (C), Deleted (D), Modified (M), Renamed (R), have their
           type (i.e. regular file, symlink, submodule, ...) changed (T), are Unmerged (U), are Unknown (X), or
           have had their pairing Broken (B). Any combination of the filter characters (including none) can be
           used. When * (All-or-none) is added to the combination, all paths are selected if there is any file
           that matches other criteria in the comparison; if there is no file that matches other criteria,
           nothing is selected.
```

**Comment by tanyaparker** [comment]
Thanks, Dave!

File: tools/hooks/hooks_utils.rb (hunk 1 above)
Comment (aoby) [reply tools/hooks/hooks_utils.rb:14]: Oh, this is the [diff-filter](https://git-scm.com/docs/git-diff#git-diff---diff-filterACDMRTUXB82308203). Ours doesn't have an equal sign so I thought it was a different option!

**Comment by davidsbailey** [comment]
> Just to be clear, this will block the content commit (via git hooks) and display an error which will cause the robo content-commit to fail and notify, right?

correct!

**Pull request merged**